### PR TITLE
add metadata example

### PR DESCRIPTION
**Pull request by bdeng-xt**
This PR added a metadata example in a docker container

### CODE REVIEW

File: Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0
+
+RUN cd /home && \
```
Comment (ihnorton) [inline Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile:3]: Please use `ADD` with the path of the example folder.

File: Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  git clone https://github.com/TileDB-Inc/TileDB-Cloud-CSharp.git -b bd/ch13349-metadata-example && \
+  dotnet publish TileDB-Cloud-CSharp/Examples_Nuget/TileDB.ExampleMetadata/metadata.csproj -c Release -o /app/example_metadata
+
+
+RUN cd /app/example_metadata 
+
+WORKDIR /app/example_metadata
```
Comment (ihnorton) [inline Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile:10]: Does it work correctly if you are in a different directory?

**Review by ihnorton — CHANGES_REQUESTED** [review]
Thanks, a few comments and questions. Please move everything under the top-level `examples/` folder, eg `examples/Docker_nuget`.

File: Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile (hunk 2 above)
Comment (bdeng-xt) [reply Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile:15]: Yes, if WORKDIR is in different direcotry. it still works

**Review by ihnorton — CHANGES_REQUESTED** [review]
Please update to use `ADD`, we want to copy and build from current directory tree instead of a new checkout inside the container.

**Pull request closed without merging**